nodejs/week2/src/backend/routes/reservations.js

```diff
@@ -0,0 +1,20 @@
+const express = require("express");
+const router = express.Router();
+const reservations = require("../data/reservations.json");
+
+router.get("/reservations",(request, response) => {
+
+        response.send(reservations);
+    });
+
+// params.. reservation with specific id
+router.get("/reservations/:id", (request, response) => {
+        const reservationId = request.params.id;
+        reservations.forEach((reservation) => {
+                if (reservation.id === parseInt(reservationId)) {
```

Comment:
What if there are no reservation with the id that the user wants to get?
You can implement an else condition here for that.